vite.cern.config.ts

```diff
@@ -19,6 +19,13 @@ export default defineConfig(async (args) => {
     config = _defineConfig
   }
 
+  if (args.mode !== 'production') {
```

Comment:
Why don't you check the `command` to be `serve`? 
Just as in the code that your rightfully removed

In the end I don't mind what you do in your config file :)

Reply:
Ah, I see. You copied our logic and the server settings, just without the https settings. You can possibly get away without the mode check even. I think we only have it to avoid the dependency on the certs when running in build mode e.g. in CI. As you remove that you can probably apply your settings unconditionally

Reply:
Removed the check. Feel free to merge.